include/neural-graphics-primitives/json_binding.h

```diff
@@ -175,6 +176,11 @@ inline void from_json(const nlohmann::json& j, NerfDataset& dataset) {
 			from_json(ji.at("rolling_shutter"), dataset.metadata[i].rolling_shutter);
 			from_json(ji.at("camera_distortion"), dataset.metadata[i].camera_distortion);
 		}
+		if (j.contains("paths")) {
+			dataset.paths.push_back(j["paths"].at(i));
```

Comment:
I recommend not using `push_back` in case `from_json` is called multiple times on the same dataset (which would result in an incorrect number of paths). Instead, above the for loop, I would do the following:

```cpp
dataset.paths.resize(dataset.n_images, "");
if (j.contains("paths")) dataset.paths = j["paths"];
```

Reply:
that's perfect!